File: 1.jl
using RollingFunctions

data = parse.(Int, split(read(joinpath(@__DIR__, "./1.txt"), String), "\n"))

sum(map((x, y) -> x > y, data[2:end], data[1:end-1]))

sw = rolling((x)->sum(x),data,3)

sum(map((x, y) -> x > y, sw[2:end], sw[1:end-1]))

File: 1.js
var fs = require('fs');
var z = require('zebras')
var array = fs.readFileSync('1.txt').toString().split("\n");

console.log(array.map((x,i)=>{ return x > array[i-1] }).reduce((a,b)=>a+b,0));

sw = z.rolling(z.sum, 3, array)

console.log(sw.map((x,i)=>{ return x > sw[i-1] }).reduce((a,b)=>a+b,0));